ui/components/app/nft-default-image/index.scss

```diff
@@ -1,7 +1,7 @@
 .nft-default {
   padding-top: 100%;
   position: relative;
-  background-image: url('/images/default_nft.png');
+  background-image: url('../../../../app/images/default_nft.png');
```

Comment:
Are these being picked up as broken by sass or stylelint? Interested how you caught this?

Reply:
Ah! So this is broken. I'll have to revert it back.

This change stems from the webpack build system I'm working on. It requires paths be relative when compiling, and it automatically finds all referenced assets and bundles them into the final build. Our gulp build process doesn't do that.

I'll revert this change and rethink how to make it compatible with both build systems later.